rest_framework/schemas/openapi.py

```diff
@@ -39,16 +40,21 @@ def get_schema(self, request=None, public=False):
         Generate a OpenAPI schema.
         """
         self._initialise_endpoints()
+        components_schemas = {}
 
         # Iterate endpoints generating per method path operations.
-        # TODO: …and reference components.
         paths = {}
         _, view_endpoints = self._get_paths_and_endpoints(None if public else request)
         for path, method, view in view_endpoints:
             if not self.has_view_permissions(path, method, view):
                 continue
 
             operation = view.schema.get_operation(path, method)
+            component = view.schema.get_components(path, method)
```

Comment:
name this `components` with an `s` just in case folks want to return multiple components.